_source/_posts/2021-10-05-okta-dev-plan-changes.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: blog_post
+title: Okta Developer Plan Changes
+author: ryan-stewart 
+by: internal-contributor
+communities: [.net,java,javascript,go,php,python,ruby,devops,announcement]
+description: "This announces how we are streamlining and clarifying our developer product offering in light of the Auth0 acquisition. We are sunsetting Okta's Developer Advanced plan, and providing Auth0 plans for developers seeking a fully self-service experience."
```

Comment:
This description might be a bit lengthy. It's recommended descriptions are under 150 characters. https://moz.com/learn/seo/meta-description

Reply:
```suggestion
description: "In light of the Auth0 acquisition, Okta is streamlining our developer product offering by retiring Okta's Developer Advanced plan, and providing Auth0 plans for developers seeking a fully self-service experience."
```